From: =?UTF-8?q?Carl=20F=C3=BCrstenberg?= <carl@excito.com>
Date: Fri, 21 Oct 2011 15:36:09 +0200
Subject: Enable missing modules to make Strongswan work

the modules CONFIG_XFRM_USER and CONFIG_NET_KEY needs to be enabled for
Strongswan support
---
 arch/arm/configs/bubba3_defconfig       |    2 ++
 arch/powerpc/configs/bubbatwo_defconfig |    2 ++
 2 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/arch/arm/configs/bubba3_defconfig b/arch/arm/configs/bubba3_defconfig
index a85a07b..f551910 100644
--- a/arch/arm/configs/bubba3_defconfig
+++ b/arch/arm/configs/bubba3_defconfig
@@ -30,6 +30,8 @@ CONFIG_PM_RUNTIME=y
 CONFIG_NET=y
 CONFIG_PACKET=y
 CONFIG_UNIX=y
+CONFIG_XFRM_USER=y
+CONFIG_NET_KEY=y
 CONFIG_INET=y
 CONFIG_IP_MULTICAST=y
 CONFIG_IP_PNP=y
diff --git a/arch/powerpc/configs/bubbatwo_defconfig b/arch/powerpc/configs/bubbatwo_defconfig
index 2bd71cf..46f6778 100644
--- a/arch/powerpc/configs/bubbatwo_defconfig
+++ b/arch/powerpc/configs/bubbatwo_defconfig
@@ -31,6 +31,8 @@ CONFIG_PCI=y
 CONFIG_NET=y
 CONFIG_PACKET=y
 CONFIG_UNIX=y
+CONFIG_XFRM_USER=y
+CONFIG_NET_KEY=y
 CONFIG_INET=y
 CONFIG_IP_MULTICAST=y
 CONFIG_IP_PNP=y
-- 
